### PR TITLE
metric(ticdc): remove sinkv1 panels

### PR DESCRIPTION
**Pull request by hi-rustin**
### What problem does this PR solve?

Issue Number: ref https://github.com/pingcap/tiflow/issues/8132

### What is changed and how it works?
- remove sinkv1 panels
- remove unified sorter panels
- remove actor panels

#### Tests 

 - No code

##### Will it cause performance regression or break compatibility?
No
##### Do you need to update user documentation, design documentation or monitoring documentation?
No
### Release note 

```release-note
None
```

### CODE REVIEW

**Comment by hi-rustin** [comment]
Now, it looks like this:
image

File: metrics/grafana/ticdc.json [hunk 1]
```diff
@@ -130,8 +130,8 @@
   "editable": true,
   "gnetId": null,
   "graphTooltip": 1,
-  "id": null,
-  "iteration": 1667966386939,
+  "id": 27,
```
Comment (amyangfei) [inline metrics/grafana/ticdc.json:133]: What does this id stand for, is it conflict with `"id":27` in L2348

Comment (hi-rustin) [reply metrics/grafana/ticdc.json:133]: I don't know what it means. But I think they are at different levels.

**Review by amyangfei — APPROVED** [review]

**Review by asddongmen — APPROVED** [review]

**Pull request merged**